### PR TITLE
add config to registrar

### LINKED ISSUE #56: Add IConfiguration to module registration
Modules should be able to easily access config settings like connection strings when they are registering services (e.g. connection string when adding a DbContext).

### PR DESCRIPTION
**Pull request by ardalis**
Should Fix #56

### CODE REVIEW

**Comment by ardalis** [comment]
I haven't tested this at all - just hoping it works. :)

**Comment by david-acm** [comment]
Added the config package for the other templates and fixed the UI and DDD modules to also accept the IConfiguration parameter

**Review by david-acm — COMMENTED** [review]
LGMT

File: working/content/modulith/Shared/Modulith.SharedKernel/Modulith.SharedKernel.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.1"/>
+        <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0"/>
```
Comment (lloydkevin) [inline working/content/modulith/Shared/Modulith.SharedKernel/Modulith.SharedKernel.csproj:11]: @david-acm Technically, you only needed `<PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />` here.

**Pull request merged**